jax/_src/util.py

```diff
@@ -216,6 +219,55 @@ def wrapper(*args, **kwargs):
 
 memoize = cache(max_size=None)
 
+_CacheInfo = namedtuple("_CacheInfo", ["hits", "misses", "maxsize", "currsize"])
```

Comment:
Maybe remove the leading underscore? Mainly because the caller of `.cache_info()` will get one of these objects, so this object isn't module-internal.